qualimap/qualimap.cwl

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+# Metadata
+label: qualimap-qc
+doc: This is qualimap CWL tool definition http://qualimap.bioinfo.cipf.es/
+
+hints:
+  ResourceRequirement:
+    ramMin: 4000
+    coresMin: 1
+  SoftwareRequirement:
+    packages:
+      qualimap:
+        version: [ "2.2.2d" ]
+        specs:
+          - https://identifiers.org/rrid/RRID:SCR_001209
+          - https://bio.tools/qualimap
+  DockerRequirement:
+    dockerPull: 'quay.io/biocontainers/qualimap:2.2.2d--1'
+requirements:
+  - class: ShellCommandRequirement
+
+# Base command
+baseCommand: [ "qualimap"]
+
+# Perform RNA-seq QC analysis on paired-end data http://qualimap.bioinfo.cipf.es/doc_html/command_line.html
+# Desired java memory size can be set using --java-mem-size option
+arguments:
+  - valueFrom: |-
+      rnaseq --paired --java-mem-size="$(inputs.javamem)"
+    shellQuote: false
+
+inputs:
+  javamem:
+    type: string
+    doc: |
+      Set desired Java heap memory size
+
+  outdir:
+    type: string
+    inputBinding:
+      prefix: "-outdir"
+    doc: |
+      Output folder for HTML report and raw data.
+
+  algo:
+    type: string
+    inputBinding:
+      prefix: "-a"
+    doc: |
+      Counting algorithm:
+      uniquely-mapped-reads(default) or proportional.
```

Comment:
If there are only two allowed values, then this should become a type: enum

Reply:
Thanks - updated in commit [5740a96](https://github.com/common-workflow-library/bio-cwl-tools/pull/78/commits/5740a9618f0156e239c655ea7b0bc85f7271cc43)